packages/node/src/lib/topics/topics.spec.ts

```diff
@@ -87,19 +88,16 @@ describe('Novu Node.js package - Topics class', () => {
     };
 
     const mockedResponse = {
-      totalCount: 1,
-      page: 0,
-      pageSize: 10,
-      data: topic,
+      data: {
```

Comment:
We use Axios implementation, so users in the response they get all the information of the response including the property `data` that includes the response of the endpoint. In this case, the Topic DTO. Therefore I thought it was better to represent in the mocks the right response users receive when using the Novu package.